scripts.d/ta/795_netmask_mismatch.sh

```diff
@@ -32,6 +32,11 @@ done
 
 if [[ ${RETURN_CODE} -eq 0 ]]; then
     echo "All Weka containers have consistent netmasks"
+else 
+    echo "Recommended Resolution: determine which of these netmasks is correct, and rectify the one with"
+    echo "the wrong configuration. If Weka needs re-configuring, this will be done with commands like"
+    echo " weka local resources --container <WEKA-CONTAINER> net remove <NIC>"
+    echo " weka local resources --container <WEKA-CONTAINER> net add    <NIC> --ips <IP-SPEC> --netmask <BIT-LENGTH>"
```

Comment:
Should we remove `--ips <IP-SPEC>`? It's more unusual than not to have to specify an IP here.